### PR TITLE
Label JSX template file added to src/components

### PR DESCRIPTION
**Pull request by JohnAKASquib**
## Description

This pull requests adds the .jsx template file for creating `<label>` components inside of _src/components/03-labels_. It follows the same structure as the `<button>` template file, so its returns an HTML `<label>` and has its attributes documented with JSDoc comments. Closes [nyc-cto/USWDS-generator/issues/59](https://github.com/nyc-cto/USWDS-generator/issues/59)

### CODE REVIEW

**Comment by MathewDavidov** [comment]
Closes https://github.com/nyc-cto/USWDS-generator/issues/59

File: src/components/03-labels/label.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ *  The first element in the document with an id matching the value of the for attribute is the labeled control for this label element, if it is a labelable element.
+ * If it is not labelable then the for attribute has no effect.
+ * If there are other elements which also match the id value, later in the document, they are not considered.
+ * @property {string} config.innerText - The text to be displayed for the label
+ * @example
+ * const config = {
+ *     class: "",
+ *     for: "submit"
+ *     innerText: "Click here to submit your form: "
+ *   }
```
Comment (mkcode) [inline src/components/03-labels/label.jsx:16]: Examples should call the function.

File: src/components/03-labels/label.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * @returns {HTMLLabelElement} The JSX HTML Label
```
Comment (mkcode) [inline src/components/03-labels/label.jsx]: It doesn't return JSX. It does return an HTMLLabelElement

**Review by mkcode — CHANGES_REQUESTED** [review]
Few changes to improve the docs

**Pull request merged**